mediagrains/tools/wrap_in_gsf.py

```diff
@@ -144,10 +146,12 @@ def wrap_audio_in_gsf():
     flow_id = args.flow_id if args.flow_id else uuid.uuid4()
     source_id = args.source_id if args.source_id else uuid.uuid4()
 
+    duration = fractions.Fraction(1, args.rate)
```

Comment:
There is no rate argument for audio and what you need instead is to calculate the duration - see below.

Audio is different from video in that the framing of audio samples (e.g. 1920 for uncompressed or 1024 in a coded AAC frame) could vary, i.e. the phase shifts. E.g. a frame could contain samples [s, s+1024) but the next frame could contain samples [s + 1500, s+1500+1024] rather than [s+1024,s+1024+1024]. The grain rate (`rate` in the constructor) would need to be interpreted differently and to avoid incorrect interpretations we set it to Fraction(0, 1), i.e. no fixed grain rate it specified.

In the 2 audio cases handled by wrap_in_gsf the audio has a fixed duration and so we can set that based on the fixed sample count and the sample rate.

So, for the CogAudioFormat.AAC case:
* Calculate the duration as `duration = fractions.Fraction(args.samples_per_grain, args.sample_rate)`
  * the args in this case are assumed to be set and not None. For the uncompressed case below defaults are allowed
* Set `duration=duration`
* Set `rate=fractions.Fraction(0, 1)`

For the uncompressed audio else case:
* Set `rate=fractions.Fraction(0, 1)`
* Change the name `grain_rate` to `duration` with the calculation as above, i.e. reverse the numerator and denominator
* Set `duration=duration`